__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require("supertest");
+const { db } = require("../db/connection");
+
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("API Topics", () => {
+  describe("200 GET topics", () => {
+    it("should return with slug and description as strings", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+            response.body.topics.forEach((topic) => {
+                expect(typeof topic.slug).toBe("string");
+                expect(typeof topic.description).toBe("string");
+            })
+        });
+    })
+    it("should return 3 topics", () => {
+      return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics.length).toBe(3)
+      })
+    })
+    it("should return the right values, checking first topic", () => {
```

Comment:
this is a bit of a brittle test, if the order of the topics came back in a different order (eg. deleting them, adding a default sort) it might not pass. This is fine for this ticket where you're just getting the topics, but I wouldn't recommend it for when you're getting back all the articles

Reply:
Thanks, I was thinking along the same lines, but figured we were reseeding the database before each test so the results should be consistent. I suppose if I add a post request above get it'll cause me issues.